render.c

```diff
@@ -194,6 +200,34 @@ void render_frame(struct swaylock_surface *surface) {
 
 	if (state->args.indicator ||
 			(upstream_show_indicator && state->auth_state != AUTH_STATE_GRACE)) {
+		// Draw indicator image
+		cairo_surface_t * image = state->indicator_image;
+		if (image) {
+			int height = state->indicator_image_height;
+			int width = state->indicator_image_height;
```

Comment:
This is probably a typo.

Reply:
This might be because we want the image to be square? It was a while ago I coded this so might be a typo